merkle-tools/src/lib.rs

```diff
@@ -1,15 +1,15 @@
-use hex::{decode, encode};
+pub type H256 = [u8; 32];
```

Comment:
Can we use the H256 from btc-types and not redefine it here?

Reply:
c86a4e7e24a7da8361b44b193bb18e83b169addb